src/components/GrampsjsPerson.js

```diff
@@ -49,16 +49,17 @@ export class GrampsjsPerson extends GrampsjsObject {
     }
     const surname = this.data.profile.name_surname || html`&hellip;`
     const suffix = this.data.profile.name_suffix || ''
+    const nick = this.data?.primary_name?.nick
     let given = this.data.profile.name_given || html`&hellip;`
     const call = this.data?.primary_name?.call
     const callIndex = call ? given.search(call) : -1
     given =
       callIndex > -1
-        ? html`${given.substr(0, callIndex)}<span class="given-name"
-              >${given.substr(callIndex, call.length)}</span
-            >${given.substr(callIndex + call.length)}`
+        ? html`${given.substring(0, callIndex)}<span class="given-name"
+              >${given.substring(callIndex, call.length)}</span
+            >${given.substring(callIndex + call.length)}`
```

Comment:
Good catch to use `substring` in place of `substr`, but on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#the_difference_between_substring_and_substr) I found this

> The two parameters of substr() are start and length, while for substring(), they are start and end.

So don't you have to change the arguments as well?